Installer fails if Kismet not pre-installed

## Description

The install script (`install/install.sh`) currently exits with an error if Kismet is not already installed on the system, requiring users to install Kismet manually before running the installer.

```
[ERROR] Kismet is not installed. Please install Kismet first.
[INFO] See: https://www.kismetwireless.net/docs/readme/installing/linux/
```

## Expected Behavior

The installer should offer to install Kismet automatically from the official Kismet repository, similar to how it handles other missing dependencies.

## Proposed Solution

Add an interactive option when Kismet is not found:

1. **Install from Kismet repository (recommended)** - Add the official Kismet apt repository and install the package
2. **Skip** - Allow manual installation and re-run

The repository installation method supports:
- Debian/Raspbian
- Ubuntu

## Affected Code

- `install/install.sh` - `preflight_checks()` function (lines 122-127)

## Additional Context

Found during fresh installation testing on Raspberry Pi OS.

--- a/install/install.sh
+++ b/install/install.sh
@@ -898,11 +898,6 @@ case "\$MODE" in
             -c ${WIFI_5GHZ}:name=${WIFI_5GHZ_NAME} \\
             -c ${WIFI_24GHZ}:name=${WIFI_24GHZ_NAME}
         ;;
-    )
-        exec kismet --no-ncurses --override  \\
-            -c ${WIFI_5GHZ}:name=${WIFI_5GHZ_NAME} \\
-            -c ${WIFI_24GHZ}:name=${WIFI_24GHZ_NAME}
-        ;;
     *)
         exec kismet --no-ncurses \\
             -c ${WIFI_5GHZ}:name=${WIFI_5GHZ_NAME} \\

